src/Tasks/InstallWindowsScheduledTask/Code/Shared/UserInputToScheduledTaskMapper.psm1

```diff
@@ -19,7 +19,7 @@ function Get-AccountCredentialsToRunScheduledTaskAs
 {
```

Comment:
Can you add these changes to `src\SharedTaskCode\UserInputToScheduledTaskMapper.psm1` and run `src\SharedTaskCode\_CopyModuleFilesToTasksSharedCodeDirectories.ps1` to copy the changes to all the tasks.